### PR TITLE
CT-2359 Hide date received legend but keep for accessibility only

### PR DESCRIPTION
**Pull request by digitalali-moj**
## Description
Hide text for legend of date recieved because it's already in heading, but keep the legend for accessibility.
 
## Self-review checklist

* [x] (1) Quick stakeholder demo done OR
* [x] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
 
### Screenshots
NEW

image

OLD

image

 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2359
 
### Deployment
N/a
 
### Manual testing instructions
GO through offender sar journey then page 4 should have no legend text

### CODE REVIEW

File: app/views/cases/offender_sar/_date_received_step.html.slim [hunk 1]
```diff
@@ -9,9 +9,9 @@
 = form_for @case, url: case_create_action(@case.object), as: :offender_sar do |f|
 
   .form-group
-    = f.gov_uk_date_field :received_date, { legend_text: t('cases.new.date_sar_received'),
-        form_hint_text: t('cases.new.date_sar_received_copy'),
-        today_button: {class: ''}}
+    = f.gov_uk_date_field :received_date, { legend_text: '',
+      form_hint_text: t('cases.new.date_sar_received_copy'),
+      today_button: {class: ''}}
```
Comment (mseedat-moj) [inline app/views/cases/offender_sar/_date_received_step.html.slim]: Formatting is a bit off. Try this:
```
= f.gov_uk_date_field :received_date, { 
    legend_text: '', 
    form_hint_text: t('cases.new.date_sar_received_copy'),
    today_button: { class: '' }
  }
```

If `class` is being set to `''` then is it still required?

**Review by mseedat-moj — CHANGES_REQUESTED** [review]
Syntax and question

File: app/views/cases/offender_sar/_date_received_step.html.slim (hunk 1 above)
Comment (digitalali-moj) [reply app/views/cases/offender_sar/_date_received_step.html.slim]: Yes might as well refactor this while visiting. Done!

Comment (digitalali-moj) [reply app/views/cases/offender_sar/_date_received_step.html.slim]: OK, so we need the class so it doesn't render it's own class (button) and change the look from link to button, even though it's a link.
Also need backslashes to allow the braces to be on separate lines

**Review by mseedat-moj — APPROVED** [review]
Thanks for the changes. The trailing slashes are unfortunate but better than having all the attributes on a single giant line 👍

**Pull request merged**